test/test.go

```diff
@@ -0,0 +1,34 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/nimblehq/mark-ic/bootstrap"
+	"github.com/nimblehq/mark-ic/helpers/log"
+
+	"github.com/gin-gonic/gin"
+)
+
+var Router *gin.Engine
+
+func SetupTestEnvironment() {
+	gin.SetMode(gin.TestMode)
+
+	setRootDir()
+
+	bootstrap.LoadConfig()
+
+	bootstrap.InitDatabase()
+}
```

Comment:
Same `not enough arguments in call to bootstrap.InitDatabase` error on my machine, I recommend fixing that before merging this PR (it's probably due to a package update that changed the way the DB is initialized)